### DISCUSSION TITLE
Html view

### DISCUSSION THREAD

**Discussion by NoeMurr** [emilk/egui#3540] [Q&A]
Hi all,

I'm trying to understand if there is a widget to render HTML.

I'm trying to create a program that needs to show markdown,  similar to the demo, but with a fully working markdown implementation with math, images and diagrams in it. 

If there is not I was thinking to implement a widget using [servo's webrenderer](https://github.com/servo/webrender) to display the compiled html. Is there something similar already done?

**Comment by div910** [comment]
Choice of the markdown rendering tool depends on the language you're working with. 
Servo's WebRender is primarily designed for rendering web content in the Servo web browser engine. If you're building a web application and want to use WebRender for rendering HTML (which can include Markdown-rendered HTML), you might need to consider integrating it within a broader web framework or application architecture.
